### PR TITLE
Add Cheetah extension

### PR DESCRIPTION
**Pull request by RongleCat**
## Description

Add Cheetah extension.
Search for a local Git project and open it with the specified application.

## Screencast

cheetah-1
cheetah-2
cheetah-3

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension and I have some feedback ready for you:

* How do I test this? Every command is just empty, I checked the README but it didn't tell me anything 😞

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Comment by pernielsentikaer** [comment]
I have it working now, thanks for the clarification 🔥 

Can't you group most of the commands into one instead, and then use the [ActionPanel](https://developers.raycast.com/api-reference/user-interface/actions#api-reference) for this? 

Did you consider that?

**Comment by RongleCat** [comment]
> 

Each command has its purpose, allowing the user to open the project with the specified application more quickly, and the ActionPanel cannot achieve the expected

File: extensions/cheetah/src/lib/constant.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { join } from "path";
+import { HOME_PATH } from "cheetah-core";
+
+export const refreshKeyword = "[refresh]";
+
+export const cachePath = join(
```
Comment (pernielsentikaer) [inline extensions/cheetah/src/lib/constant.ts:6]: You should use [environment.supportPath](https://developers.raycast.com/api-reference/environment#example) which is made for your extension's files instead.

File: extensions/cheetah/package.json [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+      "title": "Clear Cache",
+      "description": "Clear cache files.",
+      "mode": "no-view",
+      "icon": "command/clearCache.png"
+    }
+  ],
+  "preferences": [
+    {
+      "name": "workspaces",
+      "description": "The tool will search for projects in the working directory you set, with multiple directories separated by commas, Need to configure the absolute path of the directory, This directory should contain projects managed with Git.",
```
Comment (pernielsentikaer) [inline extensions/cheetah/package.json:83]: Why not support path like `~/Documents`

File: extensions/cheetah/src/lib/components/actions.tsx [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+          }}
+        />
+      </ActionPanel>
+    );
+  }
+  if (commandType === COMMAND.SET_APPLICATION) {
+    const { push } = useNavigation();
+    return (
+      <ActionPanel>
+        <Action
```
Comment (pernielsentikaer) [inline extensions/cheetah/src/lib/components/actions.tsx:34]: Missing an icon

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
I have now tested your extension again and I have some feedback ready for you 😊

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

File: extensions/cheetah/package.json (hunk 2 above)
Comment (RongleCat) [reply extensions/cheetah/package.json:83]: Originally planned to be supported in subsequent iterations, this feature has now been added, and README.md has been modified

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**